packages/api/lib/writeRecords/write-granules.js

```diff
@@ -453,28 +459,33 @@ const _writeGranuleFiles = async ({
 const _writeGranuleRecords = async (params) => {
   const {
     postgresGranuleRecord,
-    apiGranuleRecord,
     knex,
     esClient = await Search.es(),
     granuleModel,
     executionCumulusId,
     granulePgModel,
     writeConstraints = true,
   } = params;
+  let { apiGranuleRecord } = params;
   let pgGranule;
+  let pgGranuleResult;
 
   log.info('About to write granule record %j to PostgreSQL', postgresGranuleRecord);
   log.info('About to write granule record %j to DynamoDB', apiGranuleRecord);
 
   try {
     await createRejectableTransaction(knex, async (trx) => {
-      pgGranule = await _writePostgresGranuleViaTransaction({
+      pgGranuleResult = await _writePostgresGranuleViaTransaction({
         granuleRecord: postgresGranuleRecord,
         executionCumulusId,
         trx,
         granulePgModel,
         writeConstraints,
       });
+      if (pgGranuleResult.status === 'failed') {
```

Comment:
Should we log something?

Reply:
Could be fine. I was just thinking that if we're going to return undefined from a function, some kind of indicator in the logs might be nice. I haven't run it and looked at the console so if it's clear enough, that's fine.